### ISSUE TITLE
Test matches  passes when the files are't the same 

### ISSUE THREAD

**Issue by arszh** [lirantal/typeform-export-excel#13]
Test "Creating an Excel file from form matches previous sample file'"passes when the files are't the same , because `expect`  inside 'try/catch'.  Jest has  [issue #3917](https://github.com/facebook/jest/issues/3917) about this problem.

There is also a problem that the actual file from "sample.xlsx" and the exported file have different sizes.
>     actual value to equal:
>       7793
>     expected:
>       7823

## Expected Behavior

if a actual and  a expected files have different sizes then test fails

## Current Behavior

Test always passes. 

## Possible Solution

1. Remove "try/catch" because it also prevents find errors and situations where exceptions are raised.
2. Use [cvle's gist](https://github.com/facebook/jest/issues/3917#issuecomment-471007604)

## Steps to Reproduce (for bugs)

1. change `expected.size` or `actual.size` to any number
2. run test 
3. test passed
4.

## Context

I met this problem when worked on #12 

## Your Environment

- Library Version used: 1.0.6
- Node.js version (e.g. Node.js 5.4):   v12.13.1
- Operating System and version (desktop or mobile): macOS Catalina 10.15.1

**Comment by lirantal** [comment]
Oh I see. That's a good catch @arszh
Not sure how I feel about that gist workaround, doesn't seem friendly. What if we just refactor the test a bit so that the expect is simply not inside a try/catch?

**Comment by arszh** [comment]
We can declare `actual` and `expected` variables before try/catch and move down `expect(expected.size).toEqual(actual.size)` under try/catch, but I think it is not good.

In my opinion, maybe I wrong,  we can delete the try/catch altogether because:

1.  It prevents find errors and situations where exceptions are raised.
2.  Test doesn't check receiving certain exceptions.
3.  'Catch' doesn't improve information from caught exception to output it.

If important to remove created file after  this test:
1. It  will be deleted anyway when developer fix fail and  the test is passed.
2. Can use `afterAll()` to remove all created excel files after all tests.

**Comment by lirantal** [comment]
[this](https://github.com/lirantal/typeform-export-excel/blob/master/__tests__/export.test.js#L13) is the test we're referring to, right? 
```
    try {
      // eslint-disable-next-line
      const actual = fs.statSync(path.resolve('./__tests__/Fixtures/sample.xlsx'))
      // eslint-disable-next-line
      const expected = fs.statSync(path.resolve(testOutputFilename))

      expect(expected.size).toEqual(actual.size)
    } catch (e) {
    } finally {
      // eslint-disable-next-line
      fs.unlinkSync(testOutputFilename)
    }
```

in which case, the expectation is within a try/catch only for the sake of cleaning-up. In this case, how about we simply clean up in a Jest's afterAll() hook? Surely it's not as clean as with doing this within every test, but for the sake of this small lib seems not far fetched.

Looks like you are suggesting the same so 👍 from me and I'm happy to merge a PR if you have a bit of time to send it over.

**Comment by lirantal** [comment]
fixed in #18

**Comment by Harshu53** [comment]
@arszh @lirantal  Can you guys please suggest what should be fixed for below case, It is always passing : 
```
test("test", () => {
  try {
    expect(true).toBe(false);
  } catch (err) {
    console.log(err);
  }
});
```
